lib/react_on_rails/configuration.rb

```diff
@@ -11,17 +11,21 @@ def self.configuration
       logging_on_server: true,
       generator_function: false,
       trace: Rails.env.development?,
+      reload_server_js_every_request: Rails.env.development?,
```

Comment:
@justin808 although, it's technically not what's going on, it's reloading server js only if the server-bundle has been modified.